glossary.yml

```
@@ -3839,15 +3909,15 @@
   en:
     term: "Peanuts"
     def: >
-      An American comic strip by Charles M. Schulz which has inspired the names of R versions.
+      An American comic strip by Charles M. Schulz which has inspired the names of [R](#r_language) versions.
 
 
 - slug: perceptron
   en:
     term: "perceptron"
     def: >
       The simplest kind of [neural network])(#neural_network), which approximates a
-      single neuron with N binary inputs by computing a weighted sum of its inputs and
+      single neuron with $N$ binary inputs by computing a weighted sum of its inputs and
```

I don't think we have LaTeX support quite yet.

```suggestion
      single neuron with _N_ binary inputs by computing a weighted sum of its inputs and
```